### DISCUSSION TITLE
TrackDataset.plot_analogs_from_point Not calculating radius properly when changed to miles

### DISCUSSION THREAD

**Discussion by andrewfpolk** [tropycal/tropycal#200] [General]
Attempted to make a historical plot of tropical systems by changing the units of the radius to miles. 
From 40N, 60W a radius of 1111km would touch 50N.
![image](https://github.com/tropycal/tropycal/assets/33314804/a19dcb7e-f197-40bd-99dd-2bc5cabf88cd)

From 40N, 60W a radius of 690 miles should also touch 50N but it does not. 
![image](https://github.com/tropycal/tropycal/assets/33314804/f5db9f88-e905-453f-b5ad-c2ecc27c7e3e)

Code (km)
from tropycal import tracks
History = tracks.TrackDataset(basin='north_atlantic',include_btk=False)
History.plot_analogs_from_point((40.0,-60.0), radius=1111, year_range=(1970,2022))

Code (miles)
from tropycal import tracks
History = tracks.TrackDataset(basin='north_atlantic',include_btk=False)
History.plot_analogs_from_point((40.0,-60.0), radius=690, units='miles', year_range=(1970,2022))

To me it looks like a check needs to be made with changing units. Something is off I believe.
If there is a better explanation or if I am off base, would love the feedback / explanation

**Comment by tomerburg** [comment]
Thanks for bringing this to our attention! Yes, you are correct, the unit conversion to miles wasn't being done correctly. I corrected this in the develop-bugfix branch and included the correct output below (using a smaller year range for developing purposes).

A bug fix release (v1.2.2) will be released next week with a couple of important bug fixes, so this will be included in that release. If you encounter anything else in the future, you can also report anything you suspect to be a potential bug in the Issues tab, as those are checked more regularly. Thank you for reporting this again!

![image](https://github.com/tropycal/tropycal/assets/23174572/59af87a5-5f6a-424a-a577-c921173646d2)

**Reply by andrewfpolk** [reply]
Awesome! Thanks for the quick reply in the update. Love Tropycal - very useful tool!
